src/components/organisms/Table/v2/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// T Y P E S
+
+export interface BadgeProps {
+  value: string | boolean | number
+}
+
+// C O M P O N E N T S
+
+export const Badge: React.FC<BadgeProps> = (props) => {
+  const { value } = props
+  return (
+    <StyledBadge>{value.toString()}</StyledBadge>
+  )
+}
+
+const StyledBadge = styled.span`
+  background-color: var(--color-notice);
+  padding: 5px 10px;
```

Comment:
Please try to use predefined css variables to avoid inconsistant padding across components.

Reply:
yeah, good point